package.json

```diff
@@ -11,8 +11,7 @@
     "draft": "hexo new draft",
     "publish": "hexo publish",
     "hexo": "hexo",
-    "dev": "hexo server --drafts",
-    "release": "hexo clean && hexo g -d"
```

Comment:
I don't think we need this, and it was confusing for me

Reply:
yup, no longer necessary.